x-pack/plugins/apm/public/application/application.test.tsx

```diff
@@ -83,8 +84,7 @@ describe('renderApp', () => {
         getAddAlertFlyout: jest.fn(),
         getEditAlertFlyout: jest.fn(),
       },
-      data,
-      embeddable,
+      usageCollection: { reportUiCounter: () => {} },
```

Comment:
Alphabetized this and `usageCollection` from setup to start where it belongs. This makes a warning go away in Jest.